### ISSUE TITLE
Swipe up into alphabetical list

### ISSUE THREAD

**Issue by SoloRobo** [8bitPit/Niagara-Issues#2401]
### Is your feature request related to a problem?

No

### Describe the solution you'd like

I would like to be able to swipe up to naturally scroll into the alphabetical list of apps from favourites.

### Feature Request Procedure

- [X] I have read the [knowledge base article on feature requests](https://help.niagaralauncher.app/article/44-feature-requests) and followed the steps mentioned

**Comment by abhishek-junghare** [comment]
Yes, for those who prefer to open app list by swipe and not via swipe left on the alphabet list. I don't use swipe up to Niagara search, instead I would just want to open the list at the top.
